src/NuGet.Clients/NuGet.CommandLine/Commands/RestoreCommand.cs

```diff
@@ -350,6 +370,15 @@ private async Task<IReadOnlyList<RestoreSummary>> PerformNuGetV2RestoreAsync(Pac
                     packagesFolderPath,
                     restoreSummaries);
 
+                using SourceCacheContext cacheContext = new();
```

Comment:
Should we care about the `-NoCache` CLI argument, with regards to checking the vulnerability index?

Reply:
Hmm, that's interesting.

We have the same concerns in the PR restore right? Can we handle those in a separate issue/PR?

Reply:
I'd be happy to create an issue.

Reply:
I created https://github.com/NuGet/Home/issues/13261.